src/NuGetPackager/PackageCreator.cs

```diff
@@ -27,7 +27,7 @@ public PackageCreator(string packagingFolderFullPath, string nugetsFolderFullPat
                 { "authors", () => "NServiceBus Ltd" },
                 { "owners", () => "NServiceBus Ltd" },
                 { "licenseUrl", () => "http://particular.net/LicenseAgreement" },
-                { "projectUrl", () => "http://particular.net/" },
+                { "projectUrl", () => $"https://docs.particular.net/nuget/{projectName}/" },
```

Comment:
@SimonCropp I've fixed it now. Previously it was based on nuspec name which, while in theory is always the same, can differ.